### PR TITLE
Add business request form to fhir.cerner.com

### PR DESCRIPTION
**Pull request by dayouJ**
screen shot 2015-06-22 at 9 51 12 am

### CODE REVIEW

File: static/css/documentation.css [hunk 1]
```diff
@@ -898,17 +898,17 @@ a .mega-octicon {
 
 .dev-program-callout .wrapper {
   padding: 40px 0;
-  background: #160625 url(/images/callout-earth-static.png) no-repeat top right / 30%;
+  background: #FF9633 no-repeat top right / 30%;
```
Comment (MaxPhilips) [inline static/css/documentation.css]: Can you remove 'no-repeat top right /30%' since the background is just a solid color now?

**Comment by MaxPhilips** [comment]
+1 after comment is addressed

**Comment by dayouJ** [comment]
Removed 'no-repeat top right /30%'.

**Comment by dennispatterson** [comment]
Remove references to JIRA from PR title and body since this is the public github.  +1 otherwise

**Pull request merged**